packages/playground/src/views/settings.vue

```diff
@@ -231,14 +240,14 @@ export default {
     });
 
     function isCurrentTheme() {
-      return selectedTheme.value == currentTheme.value;
+      if (!currentTheme.value) return;
+      return selectedTheme.value?.includes(currentTheme.value);
```

Comment:
please change this to be a strict equal as this will return true for any theme which has the word mode in it 
![image](https://github.com/user-attachments/assets/5749086f-8c77-4881-8a57-6872b2f4b2f0)